test/testdata/lsp/completion/overloads_test.A.rbedited

```diff
@@ -0,0 +1,22 @@
+# typed: true
+# disable-stress-incremental: true
+
+class I; end
+class S; end
+class A
+  extend T::Sig
+  sig {params(x: I).void}
+  sig {params(x: S).void}
+  def my_method(x); end
+end
+
+# Not possible to see in the test, but in VS Code though these items have the
+# same name, they show up with different sigs.
+#
+# The order here doesn't particularly matter; if you're here because you made a
```

Comment:
Non-blocking and probably not for this PR: is there a way to divorce this from the order so that we don't need this comment at all? (e.g. by sorting the expectations in the test or something?)

Reply:
Caring about the order is the common case, which is what the current test harness is oriented around. see for example this test:

https://github.com/sorbet/sorbet/blob/8538079ff649facf3089186666b29d4f411cdf25/test/testdata/lsp/completion/class_and_module.rb

^ The order in this test is intentional: we're showing methods lower in the ancestry hierarchy first, because they're more likely to be relevant than say, things on Kernel. This file gets a comment because it's an exception.

We could update the test harness to be able to be able to assert "the order here doesn't matter" but so far, I've tried to just write tests that would be resilient to spurious changes. We might want to eventually.